packages/extension-ui/src/Popup/CreateAccount/SaveMnemonic.tsx

```diff
@@ -47,10 +47,12 @@ function SaveMnemonic({ className, onNextStep, onPreviousStep, seed }: Props): R
             />
           ))}
         </div>
-        <CopyToClipboard text={seed}>
+        <CopyToClipboard
+          onCopy={_onCopy}
+          text={seed}
+        >
           <Button
             className='copy-button'
-            onClick={_onCopy}
```

Comment:
The `_onCopy` function does nothing apart of showing a toast - that was really weird, considering that copying... works regardless.

Turns out `CopyToClipboard` adds the copyping handler to the first child, and also supplies a _success callback_ prop, so using `onClick` on that child to announce success was just incorrect.